caddytls/setup.go

```diff
@@ -34,6 +34,9 @@ import (
 )
 
 func init() {
+	// opt-in TLS 1.3 for Go1.12
+	os.Setenv("GODEBUG", os.Getenv("GODEBUG")+",tls13=1")
```

Comment:
@Henrocker  @mholt 
I have added this env in code.

Reply:
@mholt I can confirm that compiling Caddy with go1.12rc1 and @crvv's change successfully enables TLS 1.3 support in Caddy without the need to manually set an env variable in the OS.

Reply:
Excellent!

Can we put a TODO in the comment so we remember to revisit this later (i.e. after Go 1.13)?

Reply:
Yes, I have added the TODO.